virtual_ecosystem/models/soil/soil_model.py

```diff
@@ -315,9 +327,12 @@ def _setup(
         self.microbial_groups = microbial_groups
         self.enzyme_classes = enzyme_classes
 
-        # Store the two required hydrology constants
+        # Store the four required hydrology constants
```

Comment:
might be worth dropping the number here so you don't have to change this everytime you change the constants
```suggestion
        # Store required hydrology constants
```